Source/Core/Logging/FramesLogEvent.swift

```diff
@@ -86,8 +86,12 @@ enum FramesLogEvent: Equatable, PropertyProviding {
         case .billingFormPresented,
              .threeDSWebviewPresented:
             return [:]
-        case let .paymentFormPresented(theme, locale):
-            return [.theme: theme.rawProperties, .locale: locale.identifier].mapValues(AnyCodable.init(_:))
+        case let .paymentFormPresented(logTheme):
+            var dictionary: [Property: Any] = [.locale: Locale.current.identifier]
+            if logTheme {
+                dictionary[.theme] = Theme()
+            }
+            return dictionary.mapValues(AnyCodable.init(_:))
```

Comment:
Given the old UI and new will not be present at the same time, might be handy to drop `logTheme` altogether.

New UI theming is too complex to be logged in the same fashion as old one...

Reply:
We'll handle new UI theming logging separately. 

We probably need to revisit how theming as set, as now we have so many configurations per component, we'll get messy / oversaturated logs.

Reply:
But are we certain the old UI will not be shown alongside the new UI at all? Because then I'd be able to start removing some things that were created for logging the old UI.

Reply:
For 4.0.0 we're going to delete old UI methods, so if you want to use the old UI you'll need to use 3.5.3.